test/Section.spec.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
```

Comment:
In this case, we don't need to use `mount` as we are not testing any methods that should run on mount, shallow will do, and it's faster :)

Reply:
Oh yes, that is true. I'll fix it real quick :).